include/deal.II/fe/fe_poly_face.templates.h

```diff
@@ -238,6 +239,56 @@ FE_PolyFace<PolynomialType, dim, spacedim>::fill_fe_subface_values(
   Assert(!(fe_data.update_each & update_hessians), ExcNotImplemented());
 }
 
+
+
+template <class PolynomialType, int dim, int spacedim>
+FiniteElementDomination::Domination
+FE_PolyFace<PolynomialType, dim, spacedim>::compare_for_face_domination(
+  const FiniteElement<dim, spacedim> &fe_other) const
+{
+  return this->compare_for_domination(fe_other, 1);
```

Comment:
isn't this just the default implementation in the base class? If so, can't you just remove the function here?

Reply:
I would like to call the function below.

If we'd remove this function, wouldn't we call the `compare_for_domination()` function of `FiniteElement`, which just triggers an assertion? Or would the `override` flag ensure, that we actually call this class' member? If so, the `final` flag would be realizable @tamiko.

I'll give it a try.